src/realmApi/getRealmData.js

```diff
@@ -0,0 +1,120 @@
+import axios from 'axios';
+import { getEnv } from '../env';
+import * as log from 'loglevel';
+import jwt_decode from 'jwt-decode';
+
+const env = getEnv();
+typeof env.LOG_LEVEL === 'undefined' ? log.setDefaultLevel('debug') : log.setLevel(env.LOG_LEVEL);
+
+export const getSubGroups = async (tenantName, token) => {
+  const tokenDecoded = jwt_decode(token);
+
+  return axios
+    .get(
+      [env.OIDC_ISSUER.slice(0, 21), '/' + tokenDecoded.preferred_username, env.OIDC_ISSUER.slice(21)].join('') +
```

Comment:
why cut at 21 ?

Reply:
http://localhost:8080 the username should be inserted or i need to create another env or hardcode the url like the tests on graphql

Reply:
if in prod my oidc issuer is `http://muchmorelongerthantwntyoneandnotexactlytwnentyone/realms/default` what will happen?

Reply:
also why the preferred used name is insterted in the path???

Reply:
> also why the preferred used name is inserted in the path???

needed inside the url:
http://localhost:8080/**admin**/realms/default/groups

Reply:
> if in prod my oidc issuer is `http://muchmorelongerthantwntyoneandnotexactlytwnentyone/realms/default` what will happen?

so your suggestion is to set another env with the auth url?

Reply:
> 



> > also why the preferred used name is inserted in the path???
> 
> needed inside the url: http://localhost:8080/ **admin** /realms/default/groups

https://github.com/orchestracities/auth-management-ui/issues/80

Reply:
i think here there is a misunderstanding...

`http://localhost:8080/admin/realms/default/`

the api stays this regardless of the user, "admin" in the url path does not identify the user, but only that the api endpoint is the administration end point

Reply:
the example api call provided was 

```
curl 'http://localhost:8080/admin/realms/default/groups/${tenantId}?briefRepresentation=false'
```

if i wanted to state that the api would have needed you to pass the userid, i would have used:

```
curl 'http://localhost:8080/${userId}/realms/default/groups/${tenantId}?briefRepresentation=false'
```

this also implies that you have been lucky in the testing because the user name is admin, but everything would break if you change user

moreover, even if it was the case that the api required the user, you cannot assume that the length is always the same, you would have need to perform a search replace on a pattern.

Reply:
anyhow, yes, to cut the head to the bull, let's simplify by having an env variable `KEYCLOACK_ADMIN`